stytch/b2c/stytchapi/stytchapi.go

```diff
@@ -78,9 +109,15 @@ func NewAPIClient(env config.Env, projectID string, secret string, opts ...Optio
 // The context argument is used only during client setup and can be used to cancel client
 // creation. After the client is created and returned by this function, canceling the context has
 // no effect.
+//
+// It is highly recommended to use NewClient instead of this function since it will automatically detect the correct
+// Stytch environment from the provided projectID. If you wish to supply your own context, NewClient can be used in
+// conjunction with the WithInitializationContext option to provide the same functionality as this function. This
+// function will be deprecated in a future MAJOR release.
```

Comment:
Same here. I think we can deprecate this now or in the next minor version (if you want to give consumers at least one warning-less upgrade).